CollAction/Controllers/ProjectsController.cs

```diff
@@ -45,7 +45,7 @@ public ViewResult StartInfo()
         public IActionResult Find()
             => View();
 
-        public async Task<IActionResult> Details(int? id)
+        public async Task<IActionResult> Details(string name, int? id)
```

Comment:
Not a flaw in your PR, but if you make id non-nullable, you don't need the first NotFound() check. Doesn't need to be changed (it's not something introduced here), but it'd be a nice way to clean up.

Reply:
Fixed in c519dac